recipes/plotpy/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "plotpy" %}
+{% set version = "2.6.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/plotpy-{{ version }}.tar.gz
+  sha256: 03496d514ed59a1e6de61abfc18231ac0061d7271689d26769936ec561b0ec14
+
+build:
+  skip: true  # [py>=400 or py2k] 
```

Comment:
```suggestion
```
This isn't necessary as it's not doing anything. This package will only be built for the Python versions supported by `conda-forge` at any given point in time.